src/components/LiveSeedStats/OCRHandler/index.ts

```diff
@@ -152,12 +152,12 @@ class OCRHandler extends EventTarget {
     const res = await this.tesseractWorker.recognize(t);
 
     const i = res.data.words.findIndex(w => {
-      return w.text.toLowerCase().includes('seed');
+      return w.text.match(/^\d{7,13}$/);
```

Comment:
This is not very dependable — sometimes the OCR engine interprets some of the seeds as letters. That's why I used the `Seed:` approach because it proved better

Reply:
Maybe we can use the bounding boxes of words to get the lower text and dissect that?